examples/text_labels_and_annotations/align_axes_labels.py

```diff
@@ -0,0 +1,101 @@
+"""
+=================
+Align axes labels
+=================
```

Comment:
```suggestion
==========================
Axes labels outside floating spines
==========================
```
So far as I can tell, this is not aligning the axes labels, but rather placing them definitively outside the spines